src/main/java/org/folio/config/cache/RMAPIConfigurationCache.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.config.cache;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import org.folio.config.RMAPIConfiguration;
+import org.folio.properties.PropertyConfiguration;
+
+import java.util.concurrent.TimeUnit;
+
+public final class RMAPIConfigurationCache {
+  private static final String CACHE_KEY = "Configuration";
```

Comment:
As mentioned at standup -- RM API Configuration is tenant specific. So CACHE_KEY will need to either need to be or include the `x-okapi-tenant`